server/item/written_book.go

```diff
@@ -0,0 +1,62 @@
+package item
+
+type WrittenBook struct {
+	Title      string
+	Author     string
+	Generation uint
+	Glinted
```

Comment:
Point for this?

Reply:
Removed the glinted field and documented everything else